config/install/field.field.user.user.field_user_display_name.yml

```diff
@@ -0,0 +1,22 @@
+uuid: 82d09ddd-02da-4ef6-a6b5-1718ab447ed3
+langcode: en
+status: true
+dependencies:
+  config:
+    - field.storage.user.field_user_display_name
```

Comment:
This field name will need to change with the new conventions.

Reply:
that shouldnt even be there yet.